### ISSUE TITLE
Does not run

### ISSUE THREAD

**Issue by probonopd** [darkoverlordofdata/ShmupWarz-AppImage#1]
Does not run on `ubuntu-16.04-desktop-amd64.iso`:

```
me@host:~$ /home/me/Downloads/ShmupWarz-x86_64.AppImage 
/home/me/Downloads/ShmupWarz-x86_64.AppImage: error while loading shared libraries: libSDL2_image-2.0.so.0: cannot open shared object file: No such file or directory
```

At least the following libraries are missing from the AppImage:

- 	libSDL2_image-2.0.so.0 => not found
- 	libSDL2_mixer-2.0.so.0 => not found
- 	libSDL2_ttf-2.0.so.0 => not found
- 	libsndio.so.6.1 => not found

**Comment by darkoverlordofdata** [comment]
I've read the wiki, but I don't understand the automated testing process - I'm not finding AppImageAssistant anywhere.

Anyway, I have a spare laptop I use for testing. I did a fresh install of Trisquel7, and repoduced those errors and fixed it. 
I think I'm getting it now, so here is an updated download 
[ShmupWarz-x86_64.AppImage download](https://github.com/darkoverlordofdata/ShmupWarz-AppImage/releases/download/v0.0.1-alpha/ShmupWarz-x86_64.AppImage)

**Comment by probonopd** [comment]
AppImageAssistant is outdated. You can use the [testappimage](https://raw.githubusercontent.com/probonopd/AppImageKit/master/AppImageAssistant.AppDir/testappimage) script like this:
`testappimage /path/to/some-live.iso /path/to/ShmupWarz-x86_64.AppImage`

**Comment by darkoverlordofdata** [comment]
Hm, docker isn't compatible with either of my linux systems.

On ElementaryOS, the script fails:
Using OverlayFS because kernel is 4.x or newer
mount: wrong fs type, bad option, bad superblock on /dev/loop2,
       missing codepage or helper program, or other error

On Trisquel, Docker is available in the ppa, but the prerequisites aren't all there due to their concern for my 'freedom'- Trisquel is utra gnu.

I guess that leaves travis+docker. I don't know about that. I don't like travis, and I've never used docker. That sounds too much like work:)

So, I also did an experiment over the holiday - I asked my wife to try install chrome on linux using the .deb on their website, and she said it was as easy as in Windows. But she had trouble with doing the chmod thing to install my game...I forget that not everyone uses the command line. I think I'm going to put AppImage on hold for now - the geek in me loves it, but it may not fit my goals.

**Comment by probonopd** [comment]
I've had good successes with using Ubuntu Live ISOs.

**Comment by probonopd** [comment]
No command line needed :-)

![Executable](https://cdn-standard4.discourse.org/uploads/appimage/optimized/1X/a4889c5cb8711d6845b58135080787d2f370af35_1_500x500.gif)

http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80

**Comment by darkoverlordofdata** [comment]
I forget about doing it that way, elementaryos doesn't expose that checkbox. 

I may end up offering both appimage and apt-get installs. There is a lot of overlap. The sdl libraries have to be built from source, so I am building all my binaries on launchpad, it's way easier to use than travis, and that populates my ppa. Then I could re-package those to make the appimage.  It looks like you have some examples of that conversion.

Launchpad does a lot of quality control, it's hard make a build that is missing bits. So I am going to focus on that - testing is a bit sticky. When I try testing on a live iso, I get fusermount: mount failed: Operation not permitted cannot mount AppImage, please check your FUSE setup.

**Comment by darkoverlordofdata** [comment]
VirtualBox works. So it runs on a fresh install from ubuntu-16.04-desktop-amd64.iso: 
![screenshot from 2016-12-29 00 18 59](https://cloud.githubusercontent.com/assets/1262194/21539570/fb6a5b72-cd5c-11e6-8d66-df677d115a89.png)

**Comment by probonopd** [comment]
> I forget about doing it that way, elementaryos doesn't expose that checkbox.

Pretty sure there is a differently-worded but similar option in elementaryOS.

> I may end up offering both appimage and apt-get installs. There is a lot of overlap. The sdl libraries have to be built from source, so I am building all my binaries on launchpad, it's way easier to use than travis, and that populates my ppa. Then I could re-package those to make the appimage. It looks like you have some examples of that conversion.

Indeed, if you have everything in a ppa for trusty, then it's a matter of something like
```
app: ShmupWarz
union: true

ingredients:
  dist: trusty
  sources: 
    - deb http://us.archive.ubuntu.com/ubuntu/ trusty main universe
  ppas:
    - darkoverlordofdata/myppa
```
to convert it to an AppImage.

> When I try testing on a live iso, I get fusermount: mount failed: Operation not permitted cannot mount AppImage, please check your FUSE setup.

Which OS and how did you run it?

**Comment by darkoverlordofdata** [comment]
I have 2 systems -
Elementary Loki, kernel 4.4.0-57-generic.
Trisquel7 kernel 3.13.0-106-lowlatency

Pretty much anything I tried doing with Docker on either was failing, and I have no other requirement for it or experience with it. I decided using docker would be too much scope creep.

The only real issue with Elementary is trying to explain groups to a Windows user...
![screenshot from 2017-01-07 20 29 05](https://cloud.githubusercontent.com/assets/1262194/21747459/21dbf470-d51c-11e6-9344-c9d26a891606.png)
